spec/models/feedbook/position_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Feedbook::Position, type: :model do
+  it { should validate_presence_of(:name) }
+  it { should validate_presence_of(:job_description) }
+  it { should belong_to(:department) }
+  it 'should create a position by factory method' do
```

Comment:
In tests like this is more common to use a text like:

`creates a position by factory method` should is unnecessary.